app/modules/entities/components/layouts/category_external_ids.svelte

```diff
@@ -0,0 +1,91 @@
+<script>
+  import { categoryLabels } from '#entities/lib/entity_links'
+  import EntityClaimLink from '#entities/components/layouts/entity_claim_link.svelte'
+  import { user } from '#user/user_store'
+  import { I18n, i18n } from '#user/lib/i18n'
+  import { icon } from '#lib/utils'
+  import { getTextDirection } from '#lib/active_languages'
+  import DisplayedLinks from '#settings/components/displayed_links.svelte'
+  import Modal from '#components/modal.svelte'
+
+  export let category, categoryAvailableExternalIds
+
+  let showAllAvailableExternalIds = false
+  let showCategorySettings = false
+
+  let categoryPreferredAvailableExternalIds, displayedCategoryExternalIds
+  $: {
+    const { customProperties = [] } = $user
+    categoryPreferredAvailableExternalIds = categoryAvailableExternalIds.filter(({ property }) => customProperties.includes(property))
+  }
+  $: displayedCategoryExternalIds = showAllAvailableExternalIds ? categoryAvailableExternalIds : categoryPreferredAvailableExternalIds
+</script>
+
+{#if categoryAvailableExternalIds.length > 0}
+  <p class="category" dir={getTextDirection($user?.language)}>
+    <span class="category-label">{categoryLabels[category]}:</span>
+    {#each displayedCategoryExternalIds as { property, name, value }, i}
+      <EntityClaimLink {property} {name} {value} />{#if i !== displayedCategoryExternalIds.length - 1},{/if}
+    {/each}
+    {#if categoryPreferredAvailableExternalIds.length !== categoryAvailableExternalIds.length}
+      <button class="toggle-external-ids" on:click={() => showAllAvailableExternalIds = !showAllAvailableExternalIds}>
+        {#if showAllAvailableExternalIds}
+          {@html icon('chevron-left')}
+          {i18n('display less')}
+        {:else}
+          {@html icon('chevron-right')}
+          {#if categoryPreferredAvailableExternalIds.length === 0}
+            {i18n('display_x_links', { smart_count: categoryAvailableExternalIds.length })}
+          {:else}
+            {i18n('display_x_more', { smart_count: categoryAvailableExternalIds.length - categoryPreferredAvailableExternalIds.length })}
+          {/if}
+        {/if}
+      </button>
+    {/if}
+    <button title={i18n('Customize which links should be displayed')} on:click={() => showCategorySettings = true}>
+      {@html icon('cog')}
+    </button>
```

Comment:
What about displaying the cog icon only after clicking on "display more", to keep default display as simple as possible ?

Reply:
Done in 9f2258ccf. I was afraid it might create a bit of a hide and sick, with cases where it's impossible to access the cog button, but it those make the default display heavy with the cog.